include/onnxruntime/core/common/exceptions.h

```diff
@@ -32,6 +34,11 @@ class OnnxRuntimeException : public std::exception {
       : OnnxRuntimeException(location, nullptr, msg) {
   }
 
+  OnnxRuntimeException(const CodeLocation& location, const Status& status) noexcept
+      : OnnxRuntimeException(location, nullptr, status.ToString()) {
+    status_ = status;
```

Comment:
could be done in the initializer list?

Reply:
Changes should not be included... this was reverted when the nano-com adapter was switched to C.

Removed.